functions/log/2.1/function.json

```diff
@@ -0,0 +1,50 @@
+{
+  "description": "Log message to the application logs",
+  "label": "Log Message",
+  "category": "Debugging",
+  "icon": {
+    "name": "FileLogIcon",
+    "color": "Orange"
+  },
+  "options": [
+    {
+      "info": "Select the log severity. The severity is also reflected in the log overview.",
+      "label": "Severity",
+      "meta": {
+        "type": "Select",
+        "default": "debug",
```

Comment:
dont we want the default to be info? (that is the same level js console.log uses)

Reply:
No we want the default to be "debug" because that's what we used it for 99 out of 100 times :)
The only difference is the chip (label and color) in the application logs though.